odabot/cli.py

```diff
@@ -582,7 +585,7 @@ def update_workflows(obj, dry_run, force, loop, pattern):
                                                         frontend_name = instr_name, 
                                                         title = project['name'], 
                                                         messenger = messenger,
-                                                        roles = '' if project.get('workflow_status') == "production" else 'oda workflow developer',
+                                                        roles = '' if project.get('creative_status') == "production" else 'oda workflow developer',
```

Comment:
Yes! This is the place that I forgot. The status was set for dispatcher, but not in the frontend module.
I wonder what `project['workflow_status']` intended to be before...